graphql/typeDefs.ts

```diff
@@ -134,7 +134,7 @@ export default gql`
     isTeaching: String
     isEnrolled: String
     starsReceived: [Star]
-    starGiven: Star
+    starGiven: String
```

Comment:
Since you'll be return only the username now, wouldn't be better to name this something like `starredUser` or `starredMentor` to avoid confusion? What do you think?

Reply:
I think the name `starGiven` provides more meaning because the word `give` tells us that a user has been sent a `star` to another user

Reply:
Instead of just returning a string, maybe I should return an object containing the username + the rest of the fields in the Star type?

Reply:
Hm.. im not sure if returning an object with all that info would be ideal. Im not able to imagine a scenario where all that info would be useful on the front end after a person has already given star to another person

Reply:
Alright, that sounds reasonable enough. And yeah, I don't think the IDs would be useful for anything in the component.